src/cloud.c

```diff
@@ -63,19 +64,43 @@ static bool cloud_receive_message(const char *exchange,
 				  const char *routing_key,
 				  const char *body, void *user_data)
 {
-	/* TODO: Parser body message */
+	json_object *jso;
+	json_object *jobjkey;
+	struct l_queue *list;
+	const char *id;
+
+	jso = json_tokener_parse(body);
+	if (!jso)
+		return false;
+
+	if (!json_object_object_get_ex(jso, "id", &jobjkey))
+		return false;
+
+	if (json_object_get_type(jobjkey) != json_type_string)
+		return false;
+
+	id = json_object_get_string(jobjkey);
```

Comment:
Why getting it here if only using it at line 93?

Reply:
Because will be used in all cases